### PR TITLE
Add coverage for the ScopeItem model (implicit coverage of Tag model)

### PR DESCRIPTION
**Pull request by 0xdade**
No issues to link. Maybe my most atomic PR of the decade. I _believe_ this covers all code paths of the ScopeItem model as it exists today, and as a result also ends up covering all code paths of the Tag model (of which there are basically only two)

### CODE REVIEW

**Comment by 0xdade** [comment]
CircleCI didn't run on this one. Why does it hate me :(

File: natlas-server/tests/models/test_scope.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+10.12.13.14
+notevenanip
+"""
+
+
+def test_import_scope(app):
+    f, e, s = ScopeItem.import_scope_list(scopefile.split(), False)
+    assert len(f) == 1
+    assert len(e) == 1
+    assert len(s) == 4
```
Comment (ajacques) [inline natlas-server/tests/models/test_scope.py:57]: This is not very intuitive. What are these return values?

File: natlas-server/tests/models/test_scope.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def test_get_scope(app):
+    test_scope1 = ScopeItem(target="127.0.0.1/8", blacklist=False)
+    test_scope2 = ScopeItem(target="172.16.0.0/16", blacklist=True)
+    db.session.add(test_scope1)
+    db.session.add(test_scope2)
+    assert len(ScopeItem.getScope()) == 1
+    assert len(ScopeItem.getBlacklist()) == 1
+    db.session.rollback()
```
Comment (ajacques) [inline natlas-server/tests/models/test_scope.py:40]: What happens if the test case fails? Are the records left around in the DB?

Comment (0xdade) [reply natlas-server/tests/models/test_scope.py]: Good catch, the records would have persisted to following test cases. I've added a fixture with a `db.session.rollback()` that runs between every test case. Per the sqlalchemy docs:

```
Rollback the current transaction in progress.

If no transaction is in progress, this method is a pass-through.

This method rolls back the current transaction or nested transaction regardless of subtransactions being in effect. All subtransactions up to the first real transaction are closed. Subtransactions occur when begin() is called multiple times.
```

File: natlas-server/tests/models/test_scope.py (hunk 1 above)
Comment (0xdade) [reply natlas-server/tests/models/test_scope.py]: fail, exists, succeeds. I'm not particularly fond of this "return three lists" approach but didn't want to change logic while submitting tests.

**Pull request merged**